Framework/PythonInterface/test/python/mantid/FitFunctionsTest.py

```diff
@@ -757,6 +757,43 @@ def test_plugins_wrapped(self):
         from mantid.simpleapi import Lorentz
         self.assertEqual(str(Lorentz()), 'name=Lorentz,Scale=1,Length=50,Background=0')
 
+    def test_function_deriv_with_1DVector_Domain(self):
+        g = FunctionWrapper( "Gaussian", Height=7.5, Sigma=1.2, PeakCentre=10)
+        ws = CreateWorkspace(DataX=[0,1,2,3,4], DataY=[5,5,5,5])
+
+        dom = FunctionDomain1DVector(ws.readX(0))
+        out = func.functionDeriv(dom)
```

Comment:
Are these tests running, I can't see anywhere that defines `func`, maybe I'm missing something, but I was just trying to copy and paste the tests to run in python, but I was having no luck.

Reply:
It should be g instead of func. I will update the unit test but it also looks as if these tests are not running as it should have failed. Will investigate further...